modules/builds-identifying-image-change-triggers.adoc

```diff
@@ -12,38 +12,38 @@ As a developer, if you have image change triggers, you can identify which image
 ----
 apiVersion: build.openshift.io/v1
 kind: BuildConfig
-  metadata:
-    name: bc-ict-example
-    namespace: bc-ict-example-namespace
-  spec:
-
-    ...
-
-    triggers:
-    - imageChange:
-        from:
-          kind: ImageStreamTag
-          name: input:latest
-          namespace: bc-ict-example-namespace
-    - imageChange:
-        from:
-          kind: ImageStreamTag
-          name: input2:latest
-          namespace: bc-ict-example-namespace
-      type: ImageChange
-  status:
-    imageChangeTriggers:
-    - from:
+metadata:
+  name: bc-ict-example
+  namespace: bc-ict-example-namespace
+spec:
+
+# ...
+
+  triggers:
+  - imageChange:
+      from:
+        kind: ImageStreamTag
         name: input:latest
         namespace: bc-ict-example-namespace
-      lastTriggerTime: "2021-06-30T13:47:53Z"
-      lastTriggeredImageID: image-registry.openshift-image-registry.svc:5000/bc-ict-example-namespace/input@sha256:0f88ffbeb9d25525720bfa3524cb1bf0908b7f791057cf1acfae917b11266a69
-    - from:
+  - imageChange:
+      from:
+        kind: ImageStreamTag
         name: input2:latest
         namespace: bc-ict-example-namespace
-      lastTriggeredImageID:  image-registry.openshift-image-registry.svc:5000/bc-ict-example-namespace/input2@sha256:0f88ffbeb9d25525720bfa3524cb2ce0908b7f791057cf1acfae917b11266a69
-
-    lastVersion: 1
+    type: ImageChange
+status:
+  imageChangeTriggers:
+  - from:
+      name: input:latest
+      namespace: bc-ict-example-namespace
+    lastTriggerTime: "2021-06-30T13:47:53Z"
+    lastTriggeredImageID: image-registry.openshift-image-registry.svc:5000/bc-ict-example-namespace/input@sha256:0f88ffbeb9d25525720bfa3524cb1bf0908b7f791057cf1acfae917b11266a69
+  - from:
+      name: input2:latest
+      namespace: bc-ict-example-namespace
+    lastTriggeredImageID:  image-registry.openshift-image-registry.svc:5000/bc-ict-example-namespace/input2@sha256:0f88ffbeb9d25525720bfa3524cb2ce0908b7f791057cf1acfae917b11266a69
+
+  lastVersion: 1
```

Comment:
/cc @xiuwang

Reply:
/cc @jitendar-singh  FYI, please check this change